resources/lang/de_DE/default.po

```diff
@@ -1331,8 +1331,18 @@ msgstr ""
 "Engeltypen, welche eine Einführung benötigen, müssen von einem Supporter freigeschaltet werden "
 "(double-opt-in)."
 
-msgid "No Self Sign Up allowed"
-msgstr "Kein Selbst-Eintragen erlaubt"
+msgid "shift.self.signup"
+msgstr "Schichten selbst eintragen"
+
+msgid "shift.self.signup.allowed"
+msgstr "Schichten selbst eintragen erlaubt"
```

Comment:
To save some space in the table i would eventually just name it `Selbst eintragen erlaubt`?

Reply:
to be more specific and make less confusions I specificly did it this way